app/controllers/grades_controller.rb

```diff
@@ -40,7 +40,7 @@ def to_csv
     respond_to do |format|
       format.csv do
         send_data Grade.to_csv(params[:assignment_id].to_i),
-                  filename: "assignment-grades.csv"
+                  filename: "#{assignment.name}.csv"
```

Comment:
assignment is not defined

Reply:
We can rather use assignment id as name. It is already there in the params and wouldn't ever conflict with any existing file

Reply:
@aayush-05 the user will not understand assignment id. It is just a number. It is important to use assignment name.

Reply:
Assignment name is not there in the params and is conflicting with the existing file. What should I do? @satu0king